sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -922,16 +938,42 @@ object Radar {
      * @see [](https://radar.io/documentation/trip-tracking)
      *
      * @param[options] Configurable trip options.
+     * @param[callback] An optional callback.
      */
     @JvmStatic
-    fun startTrip(options: RadarTripOptions) {
+    fun startTrip(options: RadarTripOptions, callback: RadarTripCallback? = null) {
         if (!initialized) {
             return
         }
 
-        RadarSettings.setTripOptions(context, options)
-        apiClient.updateTrip(options, RadarTrip.RadarTripStatus.STARTED)
-        locationManager.getLocation(null)
+        apiClient.updateTrip(options, RadarTrip.RadarTripStatus.STARTED, object : RadarApiClient.RadarTripApiCallback {
+            override fun onComplete(status: RadarStatus) {
+                if (status == RadarStatus.SUCCESS) {
+                    RadarSettings.setTripOptions(context, options)
+
+                    locationManager.getLocation(null)
```

Comment:
Can you add a comment explaining why `locationManager.getLocation(null)` needs to be called here?

Reply:
Done